packages/ng/plg-banner/plg-banner.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="plgBanner">
+	<div class="plgBanner-icons">
+		<lu-icon class="plgBanner-icons-front" icon="transportRocket"></lu-icon>
+		<svg class="plgBanner-icons-back" aria-hidden="true" viewBox="0 0 32 32">
+			<path
+				d="M7.78066 27.3433C-7.15809 9.74516 2.8692 3.56387 9.33431 1.57963C15.8092 -0.408899 34.8658 -3.46454 31.634 13.4808C28.4035 30.4261 16.3229 37.4045 7.78066 27.3433Z"
+				fill="#FFE0D1"
+			/>
+		</svg>
+	</div>
+	<div class="plgBanner-content">
+		@if (bannerTitle) {
+		<div class="plgBanner-content-title">{{ bannerTitle }}</div>
+		}
+		<div class="plgBanner-content-description">
+			<ng-content></ng-content>
+		</div>
+		@if (linkLabel && linkURL) {
+		<div class="plgBanner-content-link">
+			<a class="link mod-icon u-textProduct" [href]="linkURL" target="_blank" rel="noopener noreferrer">
+				{{ linkLabel }}
+				<lu-icon icon="arrowExternal" [alt]="intl.openNewTab"></lu-icon>
```

Comment:
as we add a u-mask to 'open it on a new window" i think this is not necessary to add an "alt" on this icon

Reply:
"as we add an u-mask to 'open it on a new window'" => not here ?

Reply:
why not here ? ^^